setup.py

```diff
@@ -47,12 +42,10 @@ def is_requirement(line):
 
 setup(
     name='xblock-sql-grader',
-    version='0.1',
+    version='0.2',
     description='SQL Grader XBlock',  # TODO: write a better description.
     license='AGPLv3',
-    packages=[
-        'sql_grader',
-    ],
+    packages=find_packages(exclude=('sql_grader.tests','sql_grader.tests.*')),
```

Comment:
Because the tests aren't expected to be used by external code that imports this package. They're only internal. The `exclude` removes the tests package:

```
In [5]: find_packages(exclude=['sql_grader.tests'])
Out[5]: ['sql_grader', 'sql_grader.mixins']

In [6]: find_packages()
Out[6]: ['sql_grader', 'sql_grader.tests', 'sql_grader.mixins']
```

I fixed the syntax after the `tests/unit` → `tests/` directory change.

Reply:
I see, is that just a best practices thing? Or does something break? I'm just curious :)

Reply:
@stvstnfrd I haven't seen it written or recommended, but it made sense when I saw it: by writing `tests` here, it means that `pip` will skip the `tests` package (and directory) when installing the code, so it won't be copied to the production installation. This means that production code won't and can't use any function or files in `tests/` because the code won't even be there. This seems safe because the tests aren't needed in production.